src/lib/services/index.ts

```diff
@@ -347,6 +349,9 @@ export const createServices = (
     const inactiveUsersService = new InactiveUsersService(stores, config, {
         userService,
     });
+    const { featureLifecycleService } =
+        createFakeFeatureLifecycleService(config);
```

Comment:
we'll switch this to prod impl when we're done with controller and testing the in-memory version